### ISSUE TITLE
How to grand git permissions for my repo

### ISSUE THREAD

**Issue by extsoft** [yegor256/rultor#1153]
I added simple [rultor config](https://github.com/extsoft/insight/blob/master/.rultor.yml) in my repo. Then I tried to release and got [the message](https://github.com/extsoft/insight/issues/21#issuecomment-294347055). How can I solve this?

**Comment by 0crat** [comment]
@yegor256 please, pay attention to this issue

**Comment by amihaiemil** [comment]
@extsoft You need to invite rultor as a project collaborator with write access or, if the project is assigned to a team with write access, invite rultor in that team (then it will work with all the projects of that team)

**Comment by amihaiemil** [comment]
@original-brownbear @yegor256 Rultor is supposed to automatically accept invitations, it was implemented some time ago, but apparently it doesn't work - it's waiting on an invitation from ``decorators-squad/versioneye-api`` since yesterday. Can you login and accept it pls? :D

**Comment by extsoft** [comment]
@amihaiemil I invited Rultor, but the status is "Awaiting rultor's response".

**Comment by valery1707** [comment]
Same problem

**Comment by yegor256** [comment]
@extsoft I'll take of this right now

**Comment by yegor256** [comment]
@rultor release, tag is `1.62.4`

**Comment by yegor256** [comment]
@extsoft I guess the problem is fixed, please check

**Comment by extsoft** [comment]
@yegor256 my invitation was accepted by Rultor.

**Comment by extsoft** [comment]
Release was [successful](https://github.com/extsoft/insight/releases/tag/0.2.0), will close the issue. @valery1707 please try release too.

**Comment by 0crat** [comment]
Oops! Job `gh:yegor256/rultor#1153` was not in scope
